src/test/java/uk/gov/hmcts/ccd/v2/external/controller/CaseUserControllerTest.java

```diff
@@ -108,13 +107,19 @@ void should404WhenCaseNotFound() {
         }
 
         @Test
-        @DisplayName("should return 403 when user has limited access level")
-        void should403WhenUserLimitedAccess() {
-            when(userAuthorisation.getAccessLevel()).thenReturn(AccessLevel.GRANTED);
+        @DisplayName("should update when user has GRANTED access level")
```

Comment:
do we have a test for ALL access type? if not can you add please

Reply:
I don't think we need to create test case based on Access level... as we have removed that check completely...

Display name in this test case need to be changed to jurisdiction role to make it consistent with the changes